packages/components/src/templates/quote.stories.ts

```diff
@@ -17,174 +17,103 @@ export default {
   }
 };
 
-const img = (hidden = false) => html`
-  <img
-    class="w-16 h-16 rounded-full overflow-hidden object-cover ${hidden ? 'hidden' : ''}"
-    src="./placeholders/images/family.jpg"
-    alt="A caring father, balancing one child on his shoulder and gently cradling another in his arms."
-  />
-`;
-
-const information = (hidden = false, text = '4xl') => (hidden ? '' : html`<p class="mt-4 text-sm">${text}</p>`);
-
-/**
- * The default version of a `quote` pattern showcases a quote in an appealing layout,
```

Comment:
why did we remove all the information on the quote? as we don't have a component but only the template, I see a need for some further information on how to use it.

Reply:
The main goal is to make it a template, not a component. Quote is an example of how to use the components from the library, not a component.